domain/src/test/java/io/homeassistant/companion/android/domain/widgets/WidgetUseCaseImplSpec.kt

```diff
@@ -24,7 +24,7 @@ object WidgetUseCaseImplSpec : Spek({
                         1,
                         "domain",
                         "service",
-                        "serviceData"
+                        "{entity_id=light.dummy_light}"
```

Comment:
Ah crap, it needs quotes, right? I should just generate a json rather than trying to type it out.

Reply:
Okay, I directly copied the JSON from the logs to this.  It also contains a couple of fields besides the entity_id.  I think this is what you were looking for, but it would be nice to be able to directly call Gson on a mockk HashMap.  Unfortunately, I can't access Gson from the domain module.

Reply:
Corrected in commit 29fc233